projects/postgresql/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2016 Google Inc.
```

Comment:
This is not the license nor preamble that is typically used in the PG project.  We would typically have:

 * Portions Copyright (c) 1996-2020, PostgreSQL Global Development Group
 * Portions Copyright (c) 1994, Regents of the University of California

and everything licensed under the PG license.

Reply:
I think for the Dockerfile we should use the standard license for oss-fuzz, since it is not going to stay in the oss-fuzz repo.